tools/arm_pack_manager/index.json

```diff
@@ -480222,6 +480222,74 @@
             ]
         ],
         "sub_family": "STM32WL55"
+    },
+    "STM32WLE5JCIx": {
```

Comment:
Maybe adding also STM32WLE5CCUx for RAK3172 could be a good idea ?

Reply:
of course, that's done